api/src/database/migrations/20210905A-add-public-url.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
```

Comment:
There has to be a way we can do this without having to add an actual column to the database 🤔

Reply:
I think so too, but I'm having no luck with anything I've tried for that. Right now it creates the column but doesn't ever put any values in it since it's only populated by the transformer on read.

I'm 100% open to any suggestions on this.

Reply:
I believe I've got the other items resolved, let me know how you want to proceed on this part!